# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/libwebrtc/audio/audio_receive_stream.cc
# Commit: 26913ec8db0b
# Full Hash: 26913ec8db0b350b29145e9502136b29efdeccbc
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Reimplement ssrc/csrc stats cache. r=ng
#   
#   The cache can be a lot simpler than the old one, since libwebrtc handles
#   pruning old entries for us, and since we're now telling libwebrtc when we
#   render frames (so we don't have to make our own stats with guesses of when
# ==============================================================================

diff -r 18d3bc91b71e -r 26913ec8db0b third_party/libwebrtc/audio/audio_receive_stream.cc
--- a/third_party/libwebrtc/audio/audio_receive_stream.cc	Thu Feb 25 13:07:39 2021 -0600
+++ b/third_party/libwebrtc/audio/audio_receive_stream.cc	Mon Mar 15 12:19:57 2021 -0500
@@ -298,11 +298,6 @@
   return audio_frame_info;
 }
 
-void AudioReceiveStream::InsertAudioLevelForContributingSource(
-    const RtpPacketInfos& packet_infos) {
-  source_tracker_.OnFrameDelivered(packet_infos);
-}
-
 int AudioReceiveStream::Ssrc() const {
   return config_.rtp.remote_ssrc;
 }